libcontainer/specconv/spec_linux.go

```diff
@@ -710,7 +711,7 @@ func setupSeccomp(config *specs.LinuxSeccomp) (*configs.Seccomp, error) {
 		return nil, nil
 	}
 
-	// No default action specified, no syscalls listed, assume seccomp disabled
+	// No default action specified, no unixs listed, assume seccomp disabled
```

Comment:
I think this (and a few later entries) should keep `syscalls`.  In #1445 I replaced `sycalls.` with `unix.` (including the trailing period), and while that can still have false positives (e.g. when the word is at the end of a comment sentence), it won't accidentally do things like this.